ui/src/wizard/AzureWizard.ts

```diff
@@ -61,16 +68,27 @@ export class AzureWizard<T extends types.IActionContext> implements types.AzureW
                 if (step.shouldPrompt(this._context)) {
                     step.propertiesBeforePrompt = Object.keys(this._context).filter(k => !isNullOrUndefined(this._context[k]));
 
+                    const disposable: vscode.Disposable = ext.ui.onDidFinishPrompt((result) => {
+                        // tslint:disable-next-line: no-non-null-assertion
+                        step!.prompted = true;
+                        if (typeof result === 'string' && this._currentStepName) {
+                            this._cachedInputBoxValues[this._currentStepName] = result;
```

Comment:
Won't this break if you have duplicateSteps enabled?

Reply:
Yeah I thought about this, but I'm not sure. The only situation I've found that requires duplicate steps is my current work to make the tree item picker a wizard, but that uses quick picks instead of input boxes so it doesn't really apply.

Hard to know until we have a situation with input boxes using this, but since you commented I'm fine to disable this behavior for duplicate steps

Reply:
I mean, worst-case scenario, it just over-writes the old cached value with a new one.  I'm fine with leaving enabled 🤷‍♂️